File: Lab5/Не оптимізована версія (З вкладеним масивом)/javaScript/main.js
function check() {
   let count = 0;
   for (let i=10;i<100;i++){
       for (let j=10;j<100;j++){
            if (i===j){
                count++;
                console.log(count,": ",i*100+j);
            }
       }
   }
    document.getElementById("result").innerText = "Кількість шуканих чисел: "+count;
}

File: Lab5/Не оптимізована версія (З вкладеним масивом)/golang/main.go
package main

import "fmt"

func main() {
	var count int

	for i := 10; i < 100; i++ {
		for j := 10; j < 100; j++ {
			if i == j {
				fmt.Println(i*100 + j)
				count++
			}
		}
	}

	fmt.Println("Кількість шуканих чисел:", count)
}
